sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/EmptyArray.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal class EmptyArray<T>
+    {
+        public static readonly T[] Instance = Array.Empty<T>();
```

Comment:
`Array.Empty<T>();` already returns a static field. You don't need this type as a wrapper.

Reply:
Great, thanks! Will fix.